CMakeLists.txt

```diff
@@ -27,8 +27,11 @@ option(BEMAN_ITERATOR_INTERFACE_USE_DEDUCING_THIS
   "Make use of C++23 \"deducing this\" feature (P0847R7). Turn this off for non-conforming compilers."
   ${COMPILER_SUPPORTS_DEDUCING_THIS})
 
-option(ITERATOR_INTERFACE_ENABLE_TESTING
-       "Enable building tests and test infrastructure" ${PROJECT_IS_TOP_LEVEL})
+option(BEMAN_ITERATOR_INTERFACE_BUILD_TESTING
```

Comment:
Should we call this `BEMAN_ITERATOR_INTERFACE_BUILD_TESTS` for consistency with `BEMAN_ITERATOR_INTERFACE_BUILD_EXAMPLES`?

Reply:
Yeah, I suppose so if you agree.

Reply:
done